newIDE/app/src/LayersList/LayerRow.js

```diff
@@ -90,10 +90,11 @@ export default ({
                   },
                 ];
 
-                if(isLightingLayer) menuTemplate.splice(2, 0, {
-                  label: i18n._(t`Edit lighting layer`),
-                  click: openLightingLayerDialog,
-                });
+                if (isLightingLayer)
+                  menuTemplate.splice(2, 0, {
+                    label: i18n._(t`Edit lighting layer`),
+                    click: openLightingLayerDialog,
+                  });
```

Comment:
nitpicking: To avoid doing a splice, you can do:
```
const menuTemplate = [ 
 { ... },
 ...(isLightingLayer ? [{ label: ..., click: ... }] : [])
 { ... },
]
```

it's slightly more readable because it avoids having the "Edit lighting layer" option to be misplaced if someone update the menu template and forget to change the splice. Also you don't have to count indexes in your head :)